sky/packages/sky/lib/animation/scroll_behavior.dart

```diff
@@ -22,27 +22,28 @@ class BoundedBehavior extends ScrollBehavior {
 
   double _contentsSize;
   double get contentsSize => _contentsSize;
-  void set contentsSize (double value) {
-    if (_contentsSize != value) {
-      _contentsSize = value;
-      // TODO(ianh) now what? what if we have a simulation ongoing?
-    }
-  }
 
   double _containerSize;
   double get containerSize => _containerSize;
-  void set containerSize (double value) {
-    if (_containerSize != value) {
-      _containerSize = value;
-      // TODO(ianh) now what? what if we have a simulation ongoing?
-    }
+
+  /// Returns the new scrollOffset.
+  double updateSize({
```

Comment:
Perhaps this should be called updateExtents.  In the long run I think we should call contentsSize and containerSize contentsExtent and containerExtent.  The renames can wait.